manifests/server/config.pp

```diff
@@ -91,14 +91,12 @@
         quoted   => 'double',
       }
 
-      if ($::openldap::server::ldapi_ifs) {
```

Comment:
Didn't you want to default to `undef` for FreeBSD (using `~` in hiera I guess)?

Reply:
Sorry I didn't cleaned this part from the other PR when I splitted the original PR...